c7n/resources/glue.py

```diff
@@ -413,6 +414,47 @@ class resource_type(TypeInfo):
         cfn_type = 'AWS::Glue::SecurityConfiguration'
 
 
+@GlueSecurityConfiguration.filter_registry.register('kms-key')
+class KmsFilter(KmsRelatedFilter):
+    """
+    Filter a resource by its associcated kms key and optionally the aliasname
+    of the kms key by using 'c7n:AliasName'
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: glue-security-configuration-kms-key
+            resource: glue-security-configuration
+            filters:
+              - type: kms-key
+                key: c7n:AliasName
+                value: "^(alias/aws/)"
+                op: regex
+    """
+    schema = type_schema(
+        'kms-key',
+        key_type={'type': 'string', 'enum': [
+            's3', 'cloudwatch', 'job-bookmarks', 'all']},
+        rinherit=ValueFilter.schema,
+        **{'match-resource': {'type': 'boolean'},
+           'operator': {'enum': ['and', 'or']}})
+
+    RelatedIdsExpression = ''
+
+    def __init__(self, data, manager=None):
+        super(KmsFilter, self).__init__(data, manager)
```

Comment:
super() -> same as super(__class__, self)

Reply:
fwiw, on context, the former is py3 syntax simplification, the later py2 style, the codebase is all py3 now.